nixos/modules/hardware/rtl-sdr.nix

```diff
@@ -0,0 +1,22 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.hardware.rtl-sdr;
+
+in {
+  options.hardware.rtl-sdr = {
+    enable = lib.mkEnableOption {
```

Comment:
This should be lib.mkEnableOption '' ... ''; (it doesn't take an attrset)

Reply:
Done.